static/js/components/ShiftManagement.jsx

```diff
@@ -806,4 +810,18 @@ CurrentShiftMenu.propTypes = {
   }).isRequired,
 };
 
-export default CurrentShiftMenu;
+export function CommentOnShift(resourceType) {
```

Comment:
since it's a CommentOnShift component, is there any reason to pass in the resourceType as an argument? Won't it always be "shift"?

Reply:
True, I fixed it.